styles.css

```diff
@@ -0,0 +1,736 @@
+@import url("http://fonts.cdnfonts.com/css/cocogoose");
+@import url("http://fonts.cdnfonts.com/css/lato");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Lato, sans-serif;
+}
+
+h1,
+h2,
+h3 {
+  font-family: COCOGOOSE, sans-serif;
+}
+
+h1 {
+  color: #ec5242;
+  text-transform: uppercase;
+  font-size: 30px;
+  font-weight: normal;
+}
+
+h2 {
+  text-transform: uppercase;
+  font-size: 24px;
+  font-weight: normal;
+}
+
+h3 {
+  color: #ec5242;
+  font-weight: normal;
+}
+
+p {
+  color: #d3d3d3;
+}
+
+/* Home page */
+
+.intro-header {
+  display: none;
+}
+
+.home-intro {
+  display: flex;
+  flex-direction: column;
+  padding: 2% 6% 18%;
+  background-image: url("./images/main_bg.jpg");
+  background-size: cover;
+}
+
+/* Menu beginning */
+
+.nav-ul {
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+}
+
+.nav-ul-li-logo {
+  display: none;
+}
+
+.nav-ul-li-hamburger {
+  position: absolute;
+  display: block;
+  height: 30px;
+  width: 35px;
+}
+
+.nav-ul-li {
+  display: none;
+}
+
+.nav-ul-li-close {
+  display: none;
+}
+
+.nav-li-a {
+  text-decoration: none;
+  color: #ec5242;
+}
+
+.menu-open {
+  position: absolute;
+  top: 0;
+  left: 0;
+  height: auto;
+  width: 50%;
+  padding: 5%;
+  display: flex;
+  flex-direction: column;
+  list-style-type: none;
+  background-color: #272a31f5;
+}
```

Comment:
- [ ] Good job with the menu design 👍. However, let's improve the UI of the page by making sure that the page doesn't interfere with the menu by making the menu to be fixed and covering the page so that the user only sees the menu when they click the hamburger menu button. I have helped you fix this in the developer tools just take a look at the images below.


Your Implementation | Cindy's Design
:---:|:---:
![](https://i.imgur.com/p0GdcTX.png) | ![](https://i.imgur.com/ijyRPQR.png)